docs/Acceptable-Formulae.md

```diff
@@ -73,7 +73,8 @@ Clang is the default C/C++ compiler on macOS (and has been for a long time). Sof
 We're a package manager so we want to do things like resolve dependencies and set up applications for our users. If things require too much manual intervention then they aren't useful in a package manager.
 
 ## Stuff that requires vendored versions of Homebrew formulae
-Homebrew formulae should avoid having multiple, separate, upstream projects bundled together in a single package to avoid shipping outdated/insecure versions of software that is already a formula.
+Homebrew formulae should avoid having multiple, separate, upstream projects bundled together in a single package to avoid shipping outdated/insecure versions of software that is already a formula. [State of Software Security report](https://www.veracode.com/blog/research/announcing-state-software-security-v11-open-source-edition) by Veracode concludes
```

Comment:
```suggestion
Homebrew formulae should avoid having multiple, separate, upstream projects bundled together in a single package to avoid shipping outdated/insecure versions of software that is already a formula. The [State of Software Security report](https://www.veracode.com/blog/research/announcing-state-software-security-v11-open-source-edition) by Veracode concludes
```
or
```suggestion
Homebrew formulae should avoid having multiple, separate, upstream projects bundled together in a single package to avoid shipping outdated/insecure versions of software that is already a formula. Veracode's [State of Software Security report](https://www.veracode.com/blog/research/announcing-state-software-security-v11-open-source-edition) concludes
```
?

Reply:
I like the second version.

Reply:
I like it too, thanks for the suggestion @carlocab!